spt/features/vag_trace.cpp

```diff
@@ -0,0 +1,273 @@
+#include "stdafx.h"
+
+#if defined(SSDK2007) || defined(SSDK2013)
+#include "..\feature.hpp"
+#include "icliententity.h"
+#include "..\utils\game_detection.hpp"
+#include "ent_utils.hpp"
+#include "interfaces.hpp"
+#include "..\cvars.hpp"
+#include "tickrate.hpp"
+#include "signals.hpp"
+#include "playerio.hpp"
+
+extern IClientEntity* getPortal(const char* arg, bool verbose);
+extern IClientEntity* GetLinkedPortal(IClientEntity* portal);
```

Comment:
preferably include the header instead of manually adding these function declarations here